docs/calico-with-docker/without-docker-networking/README.md

```diff
@@ -63,13 +63,25 @@ You should see output like this on each node
 
 If you are not running in the cloud you may skip this step and jump to step 4.
 
-If you are running in the cloud, you will need to first configure an IP Pool
+If you are running in a cloud other than AWS, you will need to first configure an IP Pool
 with the `ipip` and `nat-outgoing` options.
 
 On either node:
 
     calicoctl pool add 192.168.0.0/16 --ipip --nat-outgoing
-    
+
+Instead, if you are running in AWS and all your endpoints are in the same subnet of your VPC,
+disable Source/Dest. checks on each instance.  You can disable this with the CLI,
+or in the `Networking` part of the instances' right click menus.
```

Comment:
Might just be cleaner to include this line and the one below by saying:

"You can disable this by right-clicking on the instance from the EC2 console and choosing `Change Source/Dest. Check`, or by running the following AWS CLI command for each instance:"